<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetTitle" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labIco">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labTitle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetTop" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>公司简介</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>通知信息</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>退出系统</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextOnly</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widgetMenu" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="1" colspan="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnMenu_Close">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>关闭</string>
            </property>
            <property name="text">
             <string>×</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnMenu_Max">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>全屏</string>
            </property>
            <property name="text">
             <string>□</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnMenu_Min">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>最小化</string>
            </property>
            <property name="text">
             <string>－</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetLeft" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="NavListView" name="navListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="items" stdset="0">
          <string>主界面||0||,地图监控|主界面|||,视频监控|主界面|||0xf03d,设备监控|主界面|||0xf108,系统设置||0||,防区信息|系统设置|||0xf0e8,位置调整|系统设置|||0xf060,地图编辑|系统设置|||0xf03e</string>
         </property>
         <property name="separateVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="separateHeight" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>9</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>190</y>
             <width>40</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>page0</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>150</y>
             <width>141</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>page1</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_3">
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>150</y>
             <width>81</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>page2</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_4">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>270</x>
             <y>190</y>
             <width>101</width>
             <height>41</height>
            </rect>
           </property>
           <property name="text">
            <string>page3</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_5">
          <widget class="QStackedWidget" name="page5_stack">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>19</y>
             <width>541</width>
             <height>471</height>
            </rect>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_7">
            <widget class="QLabel" name="label_8">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>20</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>当前职务</string>
             </property>
            </widget>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>40</y>
               <width>451</width>
               <height>31</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>90</y>
               <width>80</width>
               <height>18</height>
              </rect>
             </property>
             <property name="text">
              <string>变动申请</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>300</x>
               <y>90</y>
               <width>80</width>
               <height>18</height>
              </rect>
             </property>
             <property name="text">
              <string>离职申请</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_9">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>130</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>调配记录</string>
             </property>
            </widget>
            <widget class="QTableView" name="deployment">
             <property name="geometry">
              <rect>
               <x>45</x>
               <y>150</y>
               <width>471</width>
               <height>192</height>
              </rect>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_3">
             <property name="geometry">
              <rect>
               <x>230</x>
               <y>360</y>
               <width>80</width>
               <height>18</height>
              </rect>
             </property>
             <property name="text">
              <string>调配申请</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_9">
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>140</y>
               <width>101</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>pge4-2</string>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="page_10">
            <widget class="QLabel" name="label_7">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>120</y>
               <width>91</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>page4-3</string>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="page_6"/>
         <widget class="QWidget" name="page_8"/>
         <widget class="QWidget" name="page_11">
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>521</width>
             <height>461</height>
            </rect>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QPushButton" name="testpush">
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>70</y>
              <width>121</width>
              <height>41</height>
             </rect>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
           <widget class="QWidget" name="widget" native="true">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>150</y>
              <width>181</width>
              <height>211</height>
             </rect>
            </property>
            <widget class="QTextEdit" name="textEdit_7">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>80</y>
               <width>81</width>
               <height>31</height>
              </rect>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;照片&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>60</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>工号：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>120</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>合同编号：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>180</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>合同类型：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>240</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>合同状态：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>300</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>生效时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>360</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>失效时间：</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>420</y>
              <width>120</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>合同链接：</string>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>60</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_3">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>120</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_4">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>180</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_5">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>240</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_6">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>300</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_7">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>360</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
           <widget class="QTextBrowser" name="textBrowser_8">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>420</y>
              <width>161</width>
              <height>31</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="page_12"/>
         <widget class="QWidget" name="page_13">
          <widget class="QTableView" name="schedule">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>150</y>
             <width>501</width>
             <height>181</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="work_id">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>100</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>100</y>
             <width>41</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>工号</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>360</y>
             <width>69</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>√--上班</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>360</y>
             <width>69</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>×--休息</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>30</y>
             <width>101</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>排班表</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavListView</class>
   <extends>QListView</extends>
   <header>navlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
